opencodeblocks/graphics/blocks/block.py

```diff
@@ -55,15 +96,12 @@ def __init__(self, block_type: str = 'base', source: str = '', position: tuple =
         self.sockets_in = []
         self.sockets_out = []
 
-        self.title_height = 3 * title_size
-        self.title = title
+        self.title_height = 3.5 * title_size
         self.title_left_offset = 0
```

Comment:
All title related stuff could be pushed to OCBTitle.
Block title arg should be of type Union[OCBTitle, str], if str replace by an OCBTitle instance.
This will allow other title args to disapear from block init.

``` python
"""
title: Block title.
title_color: Color of the block title.
title_font: Font of the block title.
title_size: Size of the block title.
title_padding: Padding of the block title.
"""
```